Sources/NIOPerformanceTester/TCPThroughputBenchmark.swift

```diff
@@ -114,14 +116,20 @@ final class TCPThroughputBenchmark: Benchmark {
 
         let connectionEstablishedPromise: EventLoopPromise<EventLoop> = self.group.next().makePromise()
 
+        let promise = self.group.next().makePromise(of: NIOLoopBound<ServerHandler>.self)
         self.serverChannel = try ServerBootstrap(group: self.group)
             .childChannelInitializer { channel in
-                self.serverHandler = ServerHandler(connectionEstablishedPromise)
-                return channel.pipeline.addHandler(self.serverHandler)
+                channel.eventLoop.makeCompletedFuture {
+                    let serverHandler = ServerHandler(connectionEstablishedPromise, eventLoop: channel.eventLoop)
+                    promise.succeed(NIOLoopBound(serverHandler, eventLoop: channel.eventLoop))
```

Comment:
Do we want this promise succeeded if adding the handler to the pipeline failed?

Reply:
If adding the handler fails then the bootstrap will `throw` anyway so it doesn't really matter.